content/dns.md

```diff
@@ -47,17 +53,19 @@ Not all DNS providers support this feature and the implementation is usually ver
 
 *Note:* If you decide to host directly on a root domain, you will likely want to forward `www` to your root domain by setting up URL redirection (see above).
 
-For [SSL (recommended)](encryption.html), enable LetsEncrypt using our integration or upload your own certificate.
+[We recommend you use SSL](encryption.html) as a best practice. You can either enable LetsEncrypt using our integration or upload your own certificate.
 
 <h2 id="dns-propagation">DNS propagation</h2>
 
-DNS is distributed and cooperative, and it takes time for the world to see your changes.  In many countries, it usually updates within about 30 minutes, but it can take up to 24 hours or even longer in some circumstances (depending on the record's TTL). You can check your ALIAS (or ANAME or CNAME) independently of your app in the terminal by typing `dig +show www.mycompany.com`. If your `ANSWER SECTION` includes a record like this, you are in good shape:
+DNS is distributed and cooperative, and it takes time for the world to see your changes.  In many countries, it usually updates within about 30 minutes, but it can take up to 24 hours or even longer in some circumstances (depending on the record's TTL).
+
+You can check your ALIAS (or ANAME or CNAME) independently of your app in the terminal by typing `dig +show www.mycompany.com`. If your `ANSWER SECTION` includes a record like this, you are in good shape:
```

Comment:
This is tricky, since we're not saying what the output of the ALIAS record should be, exactly (i.e, maybe you can't check it directly, but you can check by the A record if it worked). I could rephrase to:

You can check if your ALIAS or ANAME or CNAME setting was successful in the terminal by typing..